exporter/elasticsearchexporter/README.md

```diff
@@ -127,10 +127,6 @@ behaviours, which may be configured throug the following settings:
              field names for span events. 
   - `fields` (optional): Configure additional fields mappings.
   - `file` (optional): Read additional field mappings from the provided YAML file.
-  - `dedup` (default=true): Try to find and remove duplicate fields/attributes
```

Comment:
We should not remove a configuration property without first deprecating it, given that this component is in `Beta` stability. Here's the documentation: https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#beta-components.

Reply:
I'll create a separate PR tomorrow with just the deprecations. I did it this way because the config does not actually have any functional benefit. Elasticsearch will reject docs with duplicate object keys, so if you disable it then the exporter won't actually be useful.